build/build-bundle.js

```diff
@@ -50,9 +49,21 @@ const DEBUG = false; // true for sourcemaps
  * @return {Promise<void>}
  */
 async function browserifyFile(entryPath, distPath) {
-  let bundle = browserify(entryPath, {debug: DEBUG});
+  let bundle = browserify(entryPath, {debug: true});
 
   bundle
+    .transform('babelify', {
+      compact: true, // Do not include superfluous whitespace characters and line terminators.
+      retainLines: true, // Keep things on the same line (looks wonky but helps with stacktraces)
+      comments: false, // Don't output comments
+      /** @param {string} comment */
+      shouldPrintComment: (comment) => comment.includes(BANNER), // Don't include @license or @preserve comments either
+      plugins: [
+        ['add-header-comment', {
```

Comment:
The banner prepending needs to be part of the browersify process, otherwise the maps will be wrong.

Reply:
doing it via a babel plugin here but could just as simply to a browserify plugin.

Reply:
oh! this adds to to _every_ file loaded in babel. I'll switch to a browserify plugin :)

Reply:
`prependify` causes off-by-one source map issues.

Reply:
Turns out it's much simpler to just add this version comment as a footer. can't break maps like that.